crates/biome_css_parser/src/syntax/property/unicode_range.rs

```diff
@@ -1,19 +1,28 @@
 use crate::lexer::{CssLexContext, CssReLexContext};
 use crate::parser::CssParser;
 use biome_css_syntax::CssSyntaxKind::{
-    CSS_BOGUS_UNICODE_RANGE_VALUE, CSS_UNICODE_CODEPOINT, CSS_UNICODE_CODEPOINT_LITERAL,
-    CSS_UNICODE_RANGE, CSS_UNICODE_RANGE_INTERVAL, CSS_UNICODE_RANGE_WILDCARD,
-    CSS_UNICODE_RANGE_WILDCARD_LITERAL,
+    self, CSS_BOGUS_UNICODE_RANGE_VALUE, CSS_DIMENSION_VALUE, CSS_NUMBER_LITERAL,
+    CSS_UNICODE_CODEPOINT, CSS_UNICODE_CODEPOINT_LITERAL, CSS_UNICODE_RANGE,
+    CSS_UNICODE_RANGE_INTERVAL, CSS_UNICODE_RANGE_WILDCARD, CSS_UNICODE_RANGE_WILDCARD_LITERAL,
 };
 use biome_css_syntax::{TextRange, T};
 use biome_parser::diagnostic::{expected_any, expected_node, ParseDiagnostic};
 use biome_parser::parsed_syntax::ParsedSyntax;
 use biome_parser::parsed_syntax::ParsedSyntax::{Absent, Present};
-use biome_parser::Parser;
+use biome_parser::{token_set, Parser, TokenSet};
+
+const UNICODE: TokenSet<CssSyntaxKind> = token_set![
+    // u+;
+    T![+],
+    // u+000;
+    CSS_NUMBER_LITERAL,
+    // u+00ff?;
+    CSS_DIMENSION_VALUE,
+];
```

Comment:
Dummy question: does not this reveal an issue in our lexer? If I understand correctly our lexer can outputs three different token sequences that are all Unicode range: `id(u) token(+)`, `id(u) number(_)` and `id(u) dimension(_)`. SHould we output a dedicated token `UnicodeRange`?

Reply:
Upon observing the lexer code, I found that the corresponding T![U+] as start, CSS_UNICODE_RANGE_WILDCARD_LITERAL CSS_UNICODE_CODEPOINT_LITERAL represents a numeric value, but we need to pass a Context to get the corresponding Token.I don't see an example of using context in other is_at methods， I'm not sure it's right to just default to set Unicode_Range_Context

Reply:
@togami2864 @denbezrukov any opinion?